releasenotes/notes/remove-schedule-sequence-a6249577da8d1c86.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The functions ``sequence`` and  ``schedule`` from the :mod:`.compiler` 
+    module have been removed following their deprecation in Qiskit 1.3.
+    These functions depend on and relate to the Pulse package which is 
+    also being removed in Qiskit 2.0.
```

Comment:
I might expand on this a bit more to explain there isn't an alternative to this because it depends on being able to translate circuits to pulse from backend definitions that is not present. But we can do this in the release round-up PR.

Reply:
Expanded in [c6f1366](https://github.com/Qiskit/qiskit/pull/13809/commits/c6f1366c68a9d5914217f42420763146a064899e), but we should probably improve it in the release PR (as most of the removal renos that relate to these large removal epics).